### ISSUE TITLE
tagsCache.cacheLibraryTags is not a function

### ISSUE THREAD

**Issue by username116** [regorxxx/Search-by-Distance-SMP#5]
Hello,

I have an error with the customizable button, used alone. It happens when I click on the button, after creating and naming it. I haven't made any settings.

I tried with a new portable installation foobar2000 v2.0 beta 12 x86 + Spider Monkey Panel v1.6.1 + the current Search-by-Distance-SMP code downloaded today (not the release).

```
Error: Spider Monkey Panel v1.6.1 (Search by Distance Customizable Button: Search by Distance Customizable Button by xxx)
tagsCache.cacheLibraryTags is not a function

File: search_bydistance.js
Line: 923, Column: 21
Stack trace:
  do_searchby_distance@search_bydistance.js:923:21
  ["Search by Distance Customizable"]<@buttons_search_bydistance_customizable.js:59:5
  themedButton/this.onClick@buttons_xxx.js:297:21
  @buttons_xxx.js:480:21
  fireEvents/</<@callbacks_xxx.js:148:37
  fireEvents/<@callbacks_xxx.js:145:30
```

![Capture d'écran 117 foobar2000](https://user-images.githubusercontent.com/108369600/197357942-aa40f5aa-2bfb-4885-81f0-5cc0fdddbd19.png)

**Comment by regorxxx** [comment]
Typo on code. Thanks! Fixed on [#ef40473](https://github.com/regorxxx/Search-by-Distance-SMP/commit/ef4047331b5cf313b50f8842f5b7e616333408ae)

**Comment by username116** [comment]
Thanks. This issue is fixed.
Maybe I'll report other issues.
(my username is Terence on the forum)
